front/components/sparkle/navigation.tsx

```diff
@@ -282,3 +284,29 @@ export const subNavigationLab = ({
 
   return nav;
 };
+
+export const subNavigationAssistant = ({
```

Comment:
should it be `subNavigationChat` ? Or should it be `subNavigationAssistantId` ?

Reply:
Done, thanks!